curriculum/challenges/english/01-responsive-web-design/basic-css-cafe-menu/part-010.md

```diff
@@ -11,10 +11,36 @@ Create an `h2` element in the `section` element and give it the text `Coffees`.
 
 # --hints--
 
-Test 1
+You should have an opening `<h2>` tag.
 
 ```js
+assert(code.match(/<h2>/i));
+```
+
+You should have a closing `</h2>` tag.
+
+```js
+assert(code.match(/<\/h2>/i));
+```
+
+You should not change your existing `section` element. Make sure you did not delete the closing tag.
 
+```js
+assert($('section').length === 1);
+```
+
+Your `h2` element should be within your `section` element.
+
+```js
+const h2 = document.querySelector('h2');
+assert(h2.parentElement.tagName === 'SECTION');
+```
+
+Your `h2` element should have the text `Coffees`.
```

Comment:
the tests are perfect. I am not sure if coffees is a word :)

Reply:
Replaced it :D